daml-lf/spec/daml-lf-1.rst

```diff
@@ -591,13 +594,9 @@ Then we can define our kinds, types, and expressions::
        |  ()                                        -- ExpUnit
        |  'True'                                    -- ExpTrue
        |  'False'                                   -- ExpFalse
-       |  'Nil' @τ                                  -- ExpListNil: Empty list
-       |  'Cons' @τ e₁ e₂                           -- ExpListCons: Cons list
-       |  'None' @τ                                 -- ExpOptionNone: Empty Option
-       |  'Some' @τ e                               -- ExpOptionSome: Non-empty Option
        |  LitInt64                                  -- ExpLitInt64: 64-bit integer literal
        |  LitNumeric                                -- ExpLitNumeric: Numeric literal
-       |  LitText                                   -- ExpLitText: UTF-8 string literal
+       |  t                                         -- ExpLitText: UTF-8 string literal
```

Comment:
Why did this change? I don’t understand why text literals here are different from other literals.

Reply:
See below.